src/lib/Utils.coffee

```diff
@@ -30,5 +30,137 @@ guessLanguageFromFilename = (filename) ->
   return 'coffeescript' if /.*\.coffee$/.test filename
   return 'javascript'
 
+isArray = (obj) ->
+  return Array.isArray(obj) if Array.isArray
+  return Object.prototype.toString.call(arg) == '[object Array]'
+
+# from http://underscorejs.org/docs/underscore.html
```

Comment:
Each function should have this comments, and it should also include mentioned of the license and copyright.